dhis-2/dhis-web-api/src/main/resources/openapi/ProgramMessageController.md

```diff
@@ -0,0 +1,61 @@
+# Send Message
+
+## Specific endpoints
+
+### `sendMessages`
+
+Send event or enrollment messages.
+
+### `getScheduledSentMessage`
+
+Get all of those scheduled messages which were sent successfully.
+
+### `getProgramMessages`
+
+Get program messages matching given query criteria.
+
+## List of supported parameters
+
+### `*.parameter.ou`
+
+Get program messages for given set of OrganisationUnits
+
+### `*.parameter.enrollment`
+
+Get program messages for given enrollment
+
+### `*.parameter.event`
+
+Get program messages for given event
+
+### `*.parameter.programInstace`
+
+Get program messages for given enrollment
```

Comment:
I think deprecated annotation is not appearing in the docs, so it would be good to add:
**DEPRECATED as of 2.41:** Use parameter `enrollment` instead.

Reply:
fixed